index.html

```diff
@@ -3,96 +3,125 @@
-            {% endif %}
-
-            <p class="role">{{card.role}}</p>
-            <p class="description line-clamp">{{card.description}}</p>
-            {% if card.social.github %}
-            <div class="socials">
-                <a href="https://github.com/{{card.social.github}}" title="GitHub"><i data-feather="globe">{{card.social.github}}</i></a>
-            </div>
-            {% endif %}
-            {% if card.social.links %}
-            {% for social-link in card.social.links %}
-            <div class="socials">
-                <a href="{{social-link[1]}}" title="{{social-link[0]}}"><i data-feather="globe">{{social-link[0]}}</i></a>
-            </div>
-            {% endfor %}
-            {% endif %}
-        </div>
-    </i>
-    {% endfor %}
-</div>
-
-<!--
-<h2>Debuggy Community Team</h2>
-<p>Everyone listed here is involved with managing and moderating the Debuggy community spaces.</p>
-
-<div class="profiles community-team">
-    <i>
-        <div class="profile-card">
-            <img class="profile-picture" src="https://cdn.discordapp.com/icons/912349224232943649/90c5433e15029bbdd96100408e23b7a9.webp" alt="test">
-            <h1>example name</h1>
-            <p class="plurality">Plural System</p>
-            <div class="plural-avatars">
-                <div class="redacted-heart"><i data-feather="heart"></i>
-                    <span class="redacted-info">Memberlist redacted by request.</span>
-                </div>
-            </div>
-            <p class="title">Community Manager</p>
-            <p class="description">We are a group of people who like making Minecraft mods.</p>
-            <div class="socials">
-                <a href="#"><i data-feather="globe"></i></a>
-                <a href="#"><i data-feather="github"></i></a>
-                <a href="#"><i data-feather="gitlab"></i></a>
-            </div>
-        </div>
-    </i>
-</div>
--->
-
-<!--
-<footer class="footer">
-    <p>Footer</p>
-</footer>
--->
-
-<script>
-    feather.replace()
-</script>
+		<h2>Contributors</h2>
+		<p>Everyone listed here has contributed to Debuggy in some way.</p>
+		<div class="profiles contributors">
+			<i>
+				<div class="profile-card">
+					<img class="profile-picture" src="https://avatars.githubusercontent.com/Cart3r1234?size=256" alt="test">
+					<h1>example name</h1>
+					<p class="plurality">Plural System</p>
+					<div class="plural-avatars">
+						<div class="redacted-heart tooltip"><i data-feather="heart"></i>
+							<span class="redacted-info tooltip-text">Memberlist redacted by request.</span>
+						</div>
+					</div>
+					<p class="title">Contributor</p>
+					<p class="description">We are a group of people who like making Minecraft mods.</p>
+					<div class="socials">
+						<a href="#"><i data-feather="globe"></i></a>
+						<a href="#"><i data-feather="github"></i></a>
+						<a href="#"><i data-feather="gitlab"></i></a>
+					</div>
+				</div>
+			</i>
+		</div>
+		-->
```

Comment:
Giant comment left behind, any reason?

Reply:
this is a leftover from when i was testing multiple team displays